RubeesEat/Pages/SplitBill.cshtml

```diff
@@ -1,11 +1,13 @@
 @page
 @inject IPersonRepository PersonRepository
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+@addTagHelper *, RubeesEat
```

Comment:
We shouldn't need these as we have imports in the layout already?

Reply:
It doesn't work if i remove them. Also we don't have the imports in the layout. Should we move them there or is it okay to leave them here?